lib/puppet/defaults.rb

```diff
@@ -1310,7 +1311,8 @@ def self.initialize_default_settings!(settings)
       end
     },
     :masterport => {
-      :default    => 8140,
+      :default    => "$serverport",
```

Comment:
why this change was needed?

Reply:
Without the `:port` setting type, this interpolation would cause the value to be a string, but now it's an integer due to the setting being munged.

It's not _needed_, but this way it's consistent with the other port settings (`ca_port` and `report_port`).